gwpy/astro/range.py

```diff
@@ -332,7 +332,8 @@ def inspiral_range_psd(psd, snr=8, mass1=1.4, mass2=1.4, horizon=False,
     )
     dist = (
         inspiral.cosmo.luminosity_distance(z_hor) if horizon else
-        range_func(f[f > 0], psd.value[f > 0], z_hor=z_hor, H=inspiral)
+        range_func(f[f > 0], psd.value[f > 0], z_hor=z_hor, H=inspiral,
+            detection_snr=snr)
```

Comment:
Please reformat this one as you have done in the other instance:

```suggestion
        range_func(
            f[f > 0],
            psd.value[f > 0],
            z_hor=z_hor,
            H=inspiral,
            detection_snr=snr,
        )
```